fabric/src/main/java/hunternif/mc/impl/atlas/mixin/fabric/TextureConfigMixin.java

```diff
@@ -0,0 +1,24 @@
+package hunternif.mc.impl.atlas.mixin.fabric;
+
+import hunternif.mc.impl.atlas.client.TextureConfig;
+import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener;
+import net.minecraft.util.Identifier;
+import org.spongepowered.asm.mixin.Mixin;
+
+import java.util.Collection;
+import java.util.List;
+
+@Mixin(TextureConfig.class)
+public abstract class TextureConfigMixin implements IdentifiableResourceReloadListener {
+
+    @Override
+    public Identifier getFabricId() {
+        return new Identifier("antiqueatlas", "textures");
```

Comment:
I'd like to reduce the redundancy with the identifier used in `getName()`. Can you add a method `getIdentifier`, which would be used for the `getFabricId()` and `getName()` methods?